.doc_gen/metadata/dynamodb_metadata.yaml

```diff
@@ -1676,6 +1680,18 @@ dynamodb_Scenario_PartiQLSingle:
             - description: Run a scenario that creates a table and runs PartiQL queries.
               snippet_tags:
                 - python.example_code.dynamodb.Scenario_PartiQLSingle
+    C++:
+      versions:
+        - sdk_version: 1
+          github: cpp/example_code/dynamodb
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - cpp.example_code.dynamodb.Scenario_PartiQL_Single

```

Comment:
I think you have the code to add snippets to the single actions for PartiQL also.